scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -264,7 +265,11 @@ def get_service_id(application_name, pipeline_name):
     try:
         data = application_name, pipeline_name
         cur_platform.execute("select s.id as service_id from service s LEFT OUTER JOIN applications a ON s.application_id = a.id LEFT OUTER JOIN service_pipeline_map spm ON spm.service_id = s.id left outer join pipeline p on spm.pipeline_id = p.id where a.name = %s and p.pipeline_name = %s", data)
-        return cur_platform.fetchone()[0]
+        serviceId = cur_platform.fetchone()
+        if serviceId is not None:
+           return serviceId[0]
+        else:
+          logging.info("service id did not find for the  application name : %s, Pipeline name :%s", application_name, pipeline_name)
```

Comment:
change it to "service id was not found for ..... "

Reply:
updated the logs